src/Skoruba.IdentityServer4.Admin.BusinessLogic.Identity/Events/Identity/GenerateInvitationTokenEvent.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Skoruba.AuditLogging.Events;
```

Comment:
Was attempting to keep with the pattern of generating events based on certain actions as seen in other areas of the code.